projects/firmware_103/src/main.c

```diff
@@ -0,0 +1,74 @@
+#include "../../../libraries/ms-common/inc/adc.h"

```

Comment:
This is strange, you shouldn't need to do this for it to work?
`#include "adc.h"`
Why doesen't this work?